### PR TITLE
Add pixel filtering parameters

### PR DESCRIPTION
**Pull request by usakhelo**
This PR addes UI for pixel filtering support.

### CODE REVIEW

File: src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.rc [hunk 1]
```diff
@@ -143,7 +150,7 @@ STYLE DS_SETFONT | WS_MINIMIZEBOX | WS_MAXIMIZEBOX | WS_POPUP | WS_VISIBLE | WS_
 CAPTION "appleseed Log"
 FONT 10, "Consolas", 400, 0, 0xCC
 BEGIN
-    CONTROL         "",IDC_EDIT_LOG,"RichEdit20W",ES_MULTILINE | ES_READONLY | ES_NUMBER | WS_BORDER | WS_VSCROLL | WS_HSCROLL | WS_TABSTOP,0,0,364,197
+    CONTROL         "",IDC_EDIT_LOG,"RichEdit20W",WS_BORDER | WS_VSCROLL | WS_HSCROLL | WS_TABSTOP | 0x2804,0,0,364,197
```
[the remark this answers is not in the document]
Comment (usakhelo) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.rc]: VS is trying to change it all the time, so I thought may I should let it do it.

File: src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp [hunk 2]
```diff
@@ -331,6 +350,14 @@ namespace
             m_spinner_tilesize->SetLimits(1, 4096, FALSE);
             m_spinner_tilesize->SetResetValue(RendererSettings::defaults().m_tile_size);
             m_spinner_tilesize->SetValue(m_settings.m_tile_size, FALSE);
+
+            // Filter size.
+            m_text_filter_size = GetICustEdit(GetDlgItem(hwnd, IDC_TEXT_FILTER_SIZE));
+            m_spinner_filter_size = GetISpinner(GetDlgItem(hwnd, IDC_SPINNER_FILTER_SIZE));
+            m_spinner_filter_size->LinkToEdit(GetDlgItem(hwnd, IDC_TEXT_FILTER_SIZE), EDITTYPE_FLOAT);
+            m_spinner_filter_size->SetLimits(0, 100, FALSE);
```
[the remark this answers is not in the document]
Comment (usakhelo) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: I think these are hard limits. I don't think we have don't limits in max.

[the remark this answers is not in the document]
Comment (usakhelo) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: I checked max and vray and they have limits 1-20. Does it make sense, should I set the same limits?

[the remark this answers is not in the document]
Comment (dictoon) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: Maybe 10 is enough. Nobody would use values larger than 3 or 4 in practice.

[the remark this answers is not in the document]
Comment (dictoon) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: Ah I see that you already set it to 20. Good enough. Thanks!

**Pull request merged**